setup.py

```diff
@@ -43,7 +44,7 @@
             'opencv-contrib-python>=4.4.0.42',
             'opencv-python>=4.4.0.42',
             'grpcio',
-            'diambra-engine',
+            'diambra-engine>2.1.0rc5', # To make sure both protobuf <3.20 (TF 2.x current limitation) and >=3.20 are supported
```

Comment:
@discordianfish here I am encountering the following error:
1) If I use `'diambra-engine>2.1.0rc5'` tests CICD does not work
1) If I use `'diambra-engine>2.1.x'` deploy CICD does not work

Locally both are fine. Could you please help out in this regard?